plugins/module_utils/foreman_helper.py

```diff
@@ -241,17 +240,6 @@ def _resource_action(self, resource, action, params):
                 action, resource, str(e)))
```

Comment:
Unrelated: is the `str()` needed? You're formatting anyway

Reply:
probably not